Makefile

```diff
@@ -91,9 +91,16 @@ DOCKERFILE_FOLDER = ./manager/docker/linux/eks-anywhere-cluster-controller
 
 IMAGE_REPO?=$(if $(AWS_ACCOUNT_ID),$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com,localhost:5000)
 IMAGE_TAG?=$(GIT_TAG)-$(shell git rev-parse HEAD)
+# Tagging with the commit hash and the branch the PR is raised against
+ifeq ($(JOB_NAME),eks-anywhere-e2e-presubmit)
```

Comment:
i dont love it, but making these env vars in the job doesnt make a lot of sense either since the pull_sha will have to be like this anyway

Reply:
That was my original thought and I'm willing to go either way, but I tested it from the Makefile because doing it in the prowjob would require another PR and approval.